src/plugins/GUITestBase/src/tests/regression_scenarios/GTTestsRegressionScenarios_7001_8000.cpp

```diff
@@ -2391,6 +2394,37 @@ GUI_TEST_CLASS_DEFINITION(test_7548) {
     }
 }
 
+GUI_TEST_CLASS_DEFINITION(test_7550) {
+    class Click101TimesScenario : public CustomScenario {
+    public:
+        void run(GUITestOpStatus& os) override {
+            for (auto notificationStack = AppContext::getMainWindow()->getNotificationStack();
+                 notificationStack->count() < 100;) {
+                if (GTUtilsMdi::activeWindow(os)->objectName() != "123 [cant_translate.fa]") {
```

Comment:
Why do you need ```if``` here? 
The window is either present or not and only one way is the correct, right?

Reply:
> Why do you need `if` here?

If the current window is not our sequence, then this window should be closed so that the sequence window is activated and it is possible to click the "Find product(s)" button.
> The window is either present or not and only one way is the correct, right?

Yes.

Reply:
So the window in the test must always be closed right?-

Reply:
What is "the window"? It is important that one window -- the "123 [cant_translate.fa]" sequence window -- be active during the entire test. How this condition is achieved does not matter (as long as it works decently): i.e. I don't care if extra windows close, switch, or don't appear at all.